src/jit/gentree.h

```diff
@@ -940,6 +940,9 @@ struct GenTree
 
 #define GTF_ARRLEN_ARR_IDX 0x80000000 // GT_ARR_LENGTH -- Length which feeds into an array index expression
 
+#define GTF_LIST_AGGREGATE 0x80000000 // GT_LIST -- Indicates that this list should be treated as an
```

Comment:
These GT_LIST nodes are need by LIR only when we have TTP_STRUCT argument that is passed on the stack.  Thy aren't needed when these are passed in registers, because  for mulri-reg args we need one PUTARG_REG per register that is passed.  For stack args we use one PURARG_STK that has as its single child a GT_LIST with one element per 8-byte sized struct field. And actually we only need use this form when we do struct promotion so that we no longer are using the TYP_STRUCT base LclVar.